src/bundle/Resources/public/scss/_tabs.scss

```diff
@@ -1,151 +1,3 @@
-.ibexa-tabs {
```

Comment:
Removed old `ez-tabs`, which was renamed to `ibexa-tabs` recently overriding new styles.